index.html

```diff
@@ -24,16 +24,15 @@
   <body>
     <div class="container">
       <div class="header">
-        <ul class="nav nav-pills pull-right">
-          <!--<iframe src="http://ghbtns.com/github-btn.html?user=skycocker&repo=chromebrew&type=watch&size=large"
-            allowtransparency="true" frameborder="0" scrolling="0" width="80" height="30"></iframe>-->
 
-          <a href="https://github.com/skycocker/chromebrew"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://s3.amazonaws.com/github/ribbons/forkme_right_gray_6d6d6d.png" alt="Fork me on GitHub"></a>
+          <a href="https://github.com/skycocker/chromebrew"><img style="position: absolute; top: 0; right: 0; border: 0;" src="forkme.png" alt="Fork me on GitHub"></a>
```

Comment:
Please remove all inline styles and reference via a class in styles.css instead.

Reply:
this is actually from the original website

Reply:
Doesn't mean it's right.  We can do better than them anyway. :)

Reply:
FINE